--- origsrc/python-caja-1.2.0/src/caja-python.c	2012-03-14 07:03:44.000000000 -0500
+++ src/python-caja-1.2.0/src/caja-python.c	2012-04-17 15:32:24.382075800 -0500
@@ -217,10 +217,12 @@ caja_python_init_python (void)
 	if (Py_IsInitialized())
 		return TRUE;
 
+#ifndef G_PLATFORM_WIN32
   	debug("g_module_open " PY_LIB_LOC "/libpython" PYTHON_VERSION "." G_MODULE_SUFFIX ".1.0");
 	libpython = g_module_open(PY_LIB_LOC "/libpython" PYTHON_VERSION "." G_MODULE_SUFFIX ".1.0", 0);
 	if (!libpython)
 		g_warning("g_module_open libpython failed: %s", g_module_error());
+#endif
 
 	debug("Py_Initialize");
 	Py_Initialize();
